templates/entry/full.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Full entry template.
+ *
+ * @package AMPConf
+ */
+
+?>
+<article id="post-<?php the_ID(); ?>" <?php post_class( 'entry entry--featured' ); ?>>
+	<figure class="entry__thumbnail">
```

Comment:
@kopepasah should all instances of `<figure class="entry__thumbnail">` be wrapped with `<?php if ( has_post_thumbnail() ) : ?>`?